leshan-bsserver-demo/src/main/java/org/eclipse/leshan/server/bootstrap/demo/LeshanBootstrapServerDemo.java

```diff
@@ -161,6 +164,10 @@ public static void main(String[] args) {
     public static void createAndStartServer(String webAddress, int webPort, String localAddress, int localPort,
             String secureLocalAddress, int secureLocalPort, String modelsFolderPath, String configFilename)
             throws Exception {
+
+        // Enable OSCORE stack (fine to do even when using DTLS or only CoAP)
+        OSCoreCoapStackFactory.useAsDefault(OscoreHandler.getContextDB());
```

Comment:
True, only endpoints that will actually use OSCORE need this. So when a particular endpoint is created it would need to be known if it will be using OSCORE. 

But yeah there is no need to do this for all endpoints. For the bootstrap server for instance it would only need OSCORE on its insecure endpoint that is listening for messages from incoming clients. (At least for now when we don't have DTLS+OSCORE.)

And updating the Californium version used would be a good step to make it possible to do this. I guess that will come with the rebase to master.